packages/app-project/stores/User/User.spec.js

```diff
@@ -57,11 +62,6 @@ describe('stores > User', function () {
     .query(true)
     .reply(200, { notifications: [] })
 
-    nock('https://graphql-stats.zooniverse.org')
-    .persist()
-    .post('/graphql')
-    .reply(200, { data: { statsCount: [] }})
-
```

Comment:
The GraphQL client uses `fetch`, so it's weird that these mocks have ever worked. 

Is `fetch` already mocked in these tests? Maybe a holdover from when they ran in Node 16 or earlier?

Reply:
🤦🏻‍♂️`fetch` is mocked with `http.request` in the tests, because these tests were originally written before it was supported in Node. 

https://github.com/zooniverse/front-end-monorepo/blob/040047945474c51cbd82919047c33569af1321c6/packages/app-project/test/create-global-document.js#L63

Reply:
See https://github.com/zooniverse/front-end-monorepo/pull/2686

Reply:
Ah-ha! That makes sense now. Thanks!

Reply:
After removing `node-fetch` from the testing global doc, this line https://github.com/zooniverse/front-end-monorepo/blob/your-stats-tests-refactor/packages/app-project/src/shared/components/SubjectPicker/helpers/fetchSubjects.js#L16-L17 is (expectedly) causing some tests to fail in fetchStatuses.spec and SubjectPicker.spec, looking into...

Reply:
Ooooh. Those requests won't be intercepted any more, if the tests are using `nock`. They'll need to be tested with `undici` in Node 18+. Personally, I would leave that change out of this PR. 

Maybe add a comment to the test setup, that `node-fetch` is being used in the tests so that `fetch` can be tested with `nock`.

Reply:
Yep, I agree, I was starting to refactor and definitely slipped out of scope (of these YourStats related changes).